biome.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.0.0/schema.json",
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
```

Comment:
Uses installed biome schema to avoid consuming the [one published](https://biomejs.dev/reference/configuration/#schema) by them.